taxonomies/totransfer.txt

```diff
@@ -0,0 +1,11140 @@
+origins:en: en:Germany
+
+<en:German wines
+de:Mecklenburger Landwein
+protected_name_file_number:en: PGI-DE-A1290
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Badischer Landwein
+protected_name_file_number:en: PGI-DE-A1279
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Landwein der Ruwer
+protected_name_file_number:en: PGI-DE-A1288
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+
+
+<en:German wines
+de:Rheingauer Landwein
+protected_name_file_number:en: PGI-DE-A1299
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Saarländischer Landwein
+protected_name_file_number:en: PGI-DE-A1302
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Sächsischer Landwein
+protected_name_file_number:en: PGI-DE-A1303
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Taubertäler Landwein
+protected_name_file_number:en: PGI-DE-A1307
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Sachsen
+protected_name_file_number:en: PDO-DE-A1277
+protected_name_type:en: pdo
+origins:en: en:Germany
+
+<en:German wines
+de:Ahrtaler Landwein
+protected_name_file_number:en: PGI-DE-A1278
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Bayerischer Bodensee-Landwein
+protected_name_file_number:en: PGI-DE-A1280
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Landwein der Mosel
+protected_name_file_number:en: PGI-DE-A1283
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Landwein der Saar
+protected_name_file_number:en: PGI-DE-A1289
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Nahegauer Landwein
+protected_name_file_number:en: PGI-DE-A1293
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Pfälzer Landwein
+protected_name_file_number:en: PGI-DE-A1294
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Regensburger Landwein
+protected_name_file_number:en: PGI-DE-A1296
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Rheinburgen-Landwein
+protected_name_file_number:en: PGI-DE-A1298
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Rheinischer Landwein
+protected_name_file_number:en: PGI-DE-A1301
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Schwäbischer Landwein
+protected_name_file_number:en: PGI-DE-A1305
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+<en:German wines
+de:Starkenburger Landwein
+protected_name_file_number:en: PGI-DE-A1306
+protected_name_type:en: pgi
+origins:en: en:Germany
+
+
+
+
+<en:German, Austria, Switzerlandn fruit spirit
+de, de, fr,de,it:Bodensee Obstler
+protected_name_file_number:en: PGI-DE+AT+CH-02570
+protected_name_type:en: gi
+origins:en: en:Germany, Austria, Switzerland
+
+<en:German, Austria, Switzerlandn fruit spirit
```

Comment:
There are several broken parents with commas

Reply:
thanks

Reply:
@stephanegigandet this one is totransfer.txt. It's normal. It's not transferred yet.